types/weixin-app/index.d.ts

```diff
@@ -936,7 +936,7 @@ declare namespace wx {
 		/** 本地缓存中的指定的 key */
 		key: string;
 		/** 接口调用的回调函数,res = {data: key对应的内容} */
-		success(res: DataResponse): void;
+		success(res: { data: object | string | undefined }): void;
```

Comment:
I don't think `object` is a good type to use. maybe you could use `data:any`?

Reply:
As Mini Program documentation [wx.getStorage](https://developers.weixin.qq.com/miniprogram/dev/api/storage/wx.getStorage.html) response `data` should be `object` or `string` ( in fact it also could be `undefined`).

`any` means any type including `number` / `boolean` and so on and it's not fit to this situation.

Reply:
You are right. but most time we would use `res.data.xxx` which will cause an error. I do not know the best way to resolve this.

Reply:
I see. I changed `object` to `Record<string, any>` and is it OK ?